includes/free-trial/partners/class-wc-calypso-bridge-partner-square.php

```diff
@@ -0,0 +1,124 @@
+ <?php
+
+/**
+ * WC Calypso Bridge Partner Square
+ *
+ *	@since   2.3.4
+ *	@version 2.3.4
```

Comment:
Same as previous comment
```suggestion
 *	@since   x.x.x
 *	@version x.x.x
```